docs/openapi/schemas/Workflow.yml

```
@@ -0,0 +1,24 @@
+title: Workflow
+type: object
+properties:
+  instance:
+    type: array
+    items: 
+      type: string
+  definition:
+    type: array
+    items: 
+      type: string
+example:
+  {
+    '@context': ['https://w3id.org/traceability/v1'],
+    'type': ['Workflow'],
+    "workflow":{
+      "instance":[
+        "urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"
+      ],
+      "definition":[
+        "urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"
+      ]
+    }
+  }
```

```suggestion
  {
    "@context": ["https://w3id.org/traceability/v1"],
    "type": ["Workflow"],
    "workflow":
      {
        "instance": ["urn:uuid:f5fb6ce4-b0b1-41b8-89b0-331ni58b7ee0"],
        "definition": ["urn:uuid:n1552885-cc91-4bb3-91f1-5466a0be084e"],
      },
  }
```

Simple whitespace and quote consistency change request